src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -932,3 +978,87 @@ button.link {
     max-width: 200px;
     width: 100%;
 }
+
+/* Right Panel */
+
+.RoomInfo {
+    background: rgba(245, 245, 245, 0.90);
+    padding: 10px;
+}
+
+.RoomAvatar {
+    margin-top: 44px;
```

Comment:
Hmm, not sure about this. I think I prefer how it looks without this (and the other changes I suggested):

![image](https://user-images.githubusercontent.com/274386/120623944-b69af080-c44f-11eb-93b9-e430afe22e5b.png)